leshan-server-core/src/main/java/org/eclipse/leshan/server/observation/ObservationService.java

```diff
@@ -21,17 +21,9 @@
 import org.eclipse.leshan.server.client.Registration;
 
 /**
- * A registry for keeping track of observed resources implemented by LWM2M Clients.
- * 
+ * A service for keeping track of observed resources implemented by LWM2M Clients.
```

Comment:
This javadoc doesn't mention the "cancel".

Proposal:
A service keeping track observation. Can be used for finding observations and cancel them.